miniscope_io/stream_daq.py

```diff
@@ -404,10 +396,11 @@ def _buffer_to_frame(
                         continue
 
                     # push previous frame_buffer into frame_buffer queue
-                    frame_buffer_queue.put(frame_buffer)
+                    frame_buffer_queue.put((frame_buffer, header_list))
```

Comment:
Is there a reason why you want to couple these two tightly? I would instead like to have these in separate queues and separate `for` loops (pasted below) because we want to know metadata more when we don't get images.
```python
for image, header_list in exact_iter(imagearray.get, None):
```

Reply:
pasted below -> pasted the corresponding line in the current code
because I don't know if referring multiple code blocks in one comment is possible.